lib/apptentive_flutter.dart

```diff
@@ -205,25 +195,13 @@ class ApptentiveFlutter {
       "key": configuration.apptentiveKey,
       "signature": configuration.apptentiveSignature,
       "log_level": configuration.logLevel.toString(),
+      "should_inherit_theme":configuration.shouldInheritAppTheme,
       "should_encrypt_storage": configuration.shouldEncryptStorage,
       "should_sanitize_log_messages": configuration.shouldSanitizeLogMessages,
```

Comment:
None of these are supported in the new android SDK. I could 't find anything relevant in the iOS side too. @frankus can you confirm?

Reply:
We don't use any of the ones below. We do use `key`, `signature`, `log_level`, and `shouldSanitizeLogMessages` though.